appveyor.yml

```diff
@@ -15,7 +17,11 @@ build_script:
 - cmd: >-
     IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" (SET COMPILER=vs2015)
 
-    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (SET COMPILER=vs2017)
+    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" (SET COMPILER=vs2017 && SET ARM64_SUPPORTED=1)
+
+    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2019" (SET COMPILER=vs2019 && SET ARM64_SUPPORTED=1)
+
+    IF "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2019" "%VISUAL_STUDIO%\\Installer\\vs_installershell.exe" modify --productId Microsoft.VisualStudio.Product.Community --channelId VisualStudio.16.Release --add Microsoft.VisualStudio.Component.VC.Tools.ARM64_SUPPORTED --add Microsoft.VisualStudio.Component.VC.Runtimes.ARM64_SUPPORTED.Spectre --quiet
```

Comment:
Looks like search and replace picked up entries here that cause issues in the appveyor build:
`2019-06-19T06:35:29 : Error : - **Microsoft.VisualStudio.Component.VC.Tools.ARM64_SUPPORTED**: No matching workload or component found.`

Reply:
I'm glad you noticed this, because SFINAE melted my brain.